github/templates/README.md

```diff
@@ -0,0 +1,79 @@
+PROJECT NAME
```

Comment:
What about using the `#` markdown syntax for titles? It's easier to write and you don't have to care about it if you change the title.